WorkloadManagementSystem/Service/PilotsLoggingHandler.py

```diff
@@ -0,0 +1,59 @@
+""" PilotsLoggingHandler is the implementation of the PilotsLogging service
+
+    The following methods are available in the Service interface
+
+    addPilotsLogging()
+    getPilotsLogging
+    setPilotsUUIDtoIDMapping()
+    addPilotsUUID()
+    deletePilotsLogging()
+
+"""
```

Comment:
the auth_xxxxxxx inside this service should be revised according to the comment in this PR (content of ConfigTemplate.cfg)